ui/webui/resources/cr_components/certificate_manager/brave_overrides/certificate_manager_v2.ts

```diff
@@ -3,8 +3,6 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
-// @ts-nocheck TODO(petemill): Define types and remove ts-nocheck
```

Comment:
whoops sorry @emerick!

Reply:
Just a drive-by removal while I was in here, haha.